static/app/views/detectors/detail.tsx

```diff
@@ -1,25 +1,81 @@
 /* eslint-disable no-alert */
 import {Fragment} from 'react';
+import styled from '@emotion/styled';
 
 import {Button, LinkButton} from 'sentry/components/button';
+import {Flex} from 'sentry/components/container/flex';
 import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
 import {ActionsProvider} from 'sentry/components/workflowEngine/layout/actions';
 import {BreadcrumbsProvider} from 'sentry/components/workflowEngine/layout/breadcrumbs';
 import DetailLayout from 'sentry/components/workflowEngine/layout/detail';
 import {useWorkflowEngineFeatureGate} from 'sentry/components/workflowEngine/useWorkflowEngineFeatureGate';
-import {IconEdit} from 'sentry/icons';
+import {IconArrow, IconEdit} from 'sentry/icons';
 import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import getDuration from 'sentry/utils/duration/getDuration';
+import {ConnectedAutomationsList} from 'sentry/views/detectors/components/connectedAutomationList';
+import DetailsPanel from 'sentry/views/detectors/components/detailsPanel';
+import IssuesList from 'sentry/views/detectors/components/issuesList';
+
+type Priority = {
+  sensitivity: string;
+  threshold: number;
+};
+
+const priorities: Priority[] = [
+  {sensitivity: 'medium', threshold: 4},
+  {sensitivity: 'high', threshold: 10},
+];
+
+const assignee = 'admin@sentry.io';
+
+const resolve_threshold = 2600000;
 
 export default function DetectorDetail() {
   useWorkflowEngineFeatureGate({redirect: true});
 
   return (
-    <SentryDocumentTitle title={t('Edit Monitor')} noSuffix>
+    <SentryDocumentTitle title={'/endpoint'} noSuffix>
       <BreadcrumbsProvider crumb={{label: t('Monitors'), to: '/monitors'}}>
         <ActionsProvider actions={<Actions />}>
           <DetailLayout>
-            <DetailLayout.Main>main</DetailLayout.Main>
-            <DetailLayout.Sidebar>sidebar</DetailLayout.Sidebar>
+            <DetailLayout.Main>
+              {/* TODO: Add chart here */}
+              <Flex column gap={space(1)}>
+                <strong>{t('Ongoing Issues')}</strong>
```

Comment:
Realized I should probably change these to `<h4>`, pls ignore for now

Reply:
I would even go a step further and extract these to a component that accepts a `title` and `children` prop.